api/src/controllers/team.js

```diff
@@ -310,6 +315,102 @@ async function updateTeam(req, res, next) {
   }
 }
 
+/**
+  * Clear the team Data from the database on /team/:team_id
+  * @param {} req request object, containing team id in the url
+  * @param {*} res response object, the deleted team document
+  * @returns 200: teamn is deleted
+  * @returns 404: team is not found
+  * @returns 400: team id is not in a valid hexadecimal format
+  */
+async function clearTeam(req, res, next) {
```

Comment:
I'm not a huge fan of calling this `clearTeam` because it's not clear to me (pun unintended) straight away what it's trying to do. Any thoughts on a better name? `resetTeamData`?